components/header/actions/src/index.scss

```diff
@@ -0,0 +1,28 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~@schibstedspain/sui-atom-button/lib/index';
+
+$bgc-header-actions: $c-primary !default;
+$c-header-actions: $c-white !default;
+$w-header-actions-icon: 42px !default;
```

Comment:
we have variables for icon size: https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_size.scss

Reply:
I wasn't sure about using them because this isn't a sui-compliant/related component. Do you think I should use it anyway?

Reply:
if it fits, yes. Even non compliant components rely on sui-theme variables